src/main/java/brave/opentracing/BraveTracer.java

```diff
@@ -76,6 +94,11 @@ public static Builder newBuilder(Tracing brave4) {
       formatToPropagation.put(Format.Builtin.TEXT_MAP, brave4.propagation());
     }
 
+    public Builder setActiveSpanSource(ActiveSpanSource activeSpanSource) {
+      this.activeSpanSource = activeSpanSource;
```

Comment:
It should be okay. For example, ThreadLocalActiveSpanSource from opentracing-util works fine, as long as you don't care whether Brave's CurrentTraceContext knows about the current active span.